app/src/main/res/layout/top_site_item.xml

```diff
@@ -27,6 +27,7 @@
             android:layout_width="40dp"
             android:layout_height="40dp"
             android:adjustViewBounds="true"
+            android:background="@color/top_site_background"
```

Comment:
What does setting the background of the favicon do that setting the drawable background of the favicon_wrapper isn't doing?

Reply:
It does nothing. Removed it